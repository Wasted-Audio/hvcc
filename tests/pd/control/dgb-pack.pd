#N canvas 99 243 450 300 12;
#X obj 177 123 pack f 7;
#X obj 177 71 loadbang;
#X msg 177 97 3;
#X obj 177 147 print;
#X connect 0 0 3 0;
#X connect 1 0 2 0;
#X connect 2 0 0 0;
