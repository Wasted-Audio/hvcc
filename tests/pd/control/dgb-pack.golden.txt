[@ 0.000] print: 3 7
